service/templates/task.html

```diff
@@ -480,6 +480,20 @@ <h3 class="section-heading">
         <a class="pointer btn btn-default" ng-class="{active:currentLogs==eventLogs}" ng-click="setCurrentLogs(eventLogs)">Event logs</a>
       </div>
     </h3>
+    <div class="row" ng-show="task.logs">
+      <div class="col-lg-12">
+        <pre>
+<span class="">External Task Logs:</span>
+<span ng-repeat="logInfo in task.logs.task"><a ng-href="[[logInfo.url]]">[[logInfo.command?logInfo.command:task.display_name]]</a></span>
+
+<span class="">External Agent Logs:</span>
+<span ng-repeat="logInfo in task.logs.agent"><a ng-href="[[logInfo.url]]">[[logInfo.command?logInfo.command:task.display_name]]</a></span>
+
+<span class="">External System Logs:</span>
+<span ng-repeat="logInfo in task.logs.system"><a ng-href="[[logInfo.url]]">[[logInfo.command?logInfo.command:task.display_name]]</a></span></pre>
```

Comment:
nit: indent differently?

Reply:
I can't change it unfortunately since the whitespace here is reflected on the page